src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -105,6 +105,7 @@
     {%- else -%}
         {%- if not valid -%}
             {%- set attr = attr|merge({class: (attr.class|default('') ~ ' form-control is-invalid')|trim}) %}
+            {% set valid = true %}
```

Comment:
What about instead just passing `true` in the `form_widget()` call below?

Reply:
Yes, that works too. Thanks